recipe/meta.yaml

```diff
@@ -38,6 +46,15 @@ requirements:
 test:
   imports:
     - nibabel
+  commands:
+    - command -v parrec2nii
```

Comment:
what does `command -v` do?  I see no mentioning of that in https://conda-forge.readthedocs.io/en/latest/testing.html

Reply:
It is similar to `which` on linux
http://www.linuxhowtos.org/manpages/1p/command.htm

Reply:
Similar to `which`, but reliably returns a non-zero exit code if the commnad doesn't exist: https://stackoverflow.com/a/677212